infrasec/tutorials/atlantis.md

```diff
@@ -0,0 +1,418 @@
+      name = "/atlantis/github/user/token"
+    }
+    ```
+
+1. Add custom environment secrets:
+
+    ```hcl
+    # from https://trussworks.slack.com/archives/CLNC1MUBS/p1615847664002200?thread_ts=1615570522.023200&cid=CLNC1MUBS
+      custom_environment_secrets = [
+        {
+          name      = "ATLANTIS_GH_TOKEN"
+          valueFrom = data.aws_ssm_parameter.github_user_token.name
+        },
+        # {
+        #   name      = "ATLANTIS_GH_WEBHOOK_SECRET"
+        #   valueFrom = local.webhook_ssm_parameter_name
+        # },
+      ]
+    ```
+
+1. Submit a PR, get approval, and `terraform apply` the code.
+
+## Step 4: Configure Your Webhook
+
+The Atlantis module conveniently creates a GitHub webhook for you, and the documentation on this is fairly good. Follow the Atlantis documentation on [how to configure that webhook](https://www.runatlantis.io/docs/configuring-webhooks.html#github-github-enterprise).
+
+## Step 5: Connect the ALB Logs Bucket 🪣
+
+Of all the things this module does, it **does not** create a logs bucket. The module itself has the expectation we will either create a new bucket or connect an existing one. In our example, we'll connect an existing logs bucket and ensure our permissions are correct.
+
+1. Locate the existing logs bucket, presumably created using the `trussworks/logs/aws` module. We're just going to put logs in the bucket that already exists for the account, in our case `transcom-gov-milmove-exp-aws-logs`.
+
+    Once we find the code that creates the bucket, we need to give Atlantis permission to add our logs to the bucket. We do this by adding our chosen logs bucket prefix (we're following the established pattern and using `alb/atlantis-exp` here) to the `alb_logs_prefixes` and the `nlb_logs_prefixes` lists [in our existing logs bucket](https://github.com/transcom/transcom-infrasec-gov/blob/0eeba19465584e10772a9b1d1f71fb3e87d7138c/transcom-gov-milmove-exp/admin-global/main.tf#L25-L26).
+
+    After that, we seal the deal by returning to our code and adding the code like this inside our module call:
+
+      ```hcl
+      alb_log_bucket_name             = "transcom-gov-milmove-exp-aws-logs"
+      alb_log_location_prefix         = "alb/atlantis-exp"
+      alb_logging_enabled             = true
+      alb_listener_ssl_policy_default = true
+      ```
+
+    Another example is available in the [legendary-waddle repo](https://github.com/trussworks/legendary-waddle/blob/6b63fc65c6d2b0237a7233ad36cb9356f329966b/trussworks-prod/atlantis-prod/main.tf#L54-L56)
+
+1. Submit a PR, get approval, and `terraform apply` the code.
+
+1. Log into the console to make sure the bucket stores logs in our prefix path by viewing the auto-created `ELBAccessLogTestFile` in the path we chose: `alb/atlantis-exp/`
+
+    <img src="https://github.com/trussworks/Engineering-Playbook/blob/3efe6ea02ed010f3db2c07921c5c8acc60406b84/infrasec/tutorials/images/atlantis_lb_bucket.png" width="450">
```

Comment:
Again, I'd say if you're using the console you're doing it wrong. But you can just say "verify the logs are showing up" and let people do it how they want -- you don't need to tell them whether to use the console or the AWS CLI. Keep your audience in mind -- they should have the knowledge to figure this out themselves.